uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -102,7 +102,67 @@ <h4 class="modal-title">
             </div>
             <ng-include ng-show="$ctrl.isLeaveType('sick')" src="pathTpl + 'request-modal/details/sickness.html'"></ng-include>
           </uib-tab>
-          <uib-tab heading="Comments"></uib-tab>
+          <uib-tab heading="Comments">
+            <div class="chr_leave-request-modal__comments-container">
```

Comment:
What do you think instead of this
```html
<!-- This is used for layour purposes (height, etc) -->
<div class="chr_leave-request-modal__comments">
  <!-- This is an individual comment block -->
  <div class="chr_leave-request-comment">
    <!-- Metadata of a comment (used for layout purposes) -->
    <div class="chr_leave-request-comment__meta">
      <span class="chr_leave-request-comment__author">John Doe</span>
      <span> - </span>
      <span class="chr_leave-request-comment__date">18/01/2015</span>
      <button class="btn btn-link">Remove</button>
    </div>
    <div class="chr_leave-request-comment__text">
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tempor sagittis mauris in pharetra. Morbi quis congue nisl. Nullam sed mi ligula. Nunc dictum metus ut nisi pulvinar pretium.
    </div>
  </div>
```
This way you have a first-level block (`.chr_leave-request-comment`) that is independent of the modal, while still having an element that is strictly related to the modal itself and that you can use for layout purposes (`.chr_leave-request-modal__comments`)

Also having a reusable, independent component like `.chr_leave-request-comment` allows you to break it down into different elements like `.chr_leave-request-comment__author` that you can then use, if needed, for specific stylings